store/index.js

```diff
@@ -137,6 +142,24 @@ export const getters = {
   },
   isSoldOut: state => {
     return isSoldOut(state);
+  },
+  precomputeEscrow: state => ({ idx, KH, KNFT }) => {
+    // This should only be used for client-side confirmation or recovery. For actual
+    // predictedAddress derivation, use the on-chain contract function.
+    throw "XXX: This isn't done yet."; // Outputs don't match up yet
```

Comment:
@mveytsman I'm losing steam on figuring out why this isn't matching up to the on-chain version, could I borrow a fresh pair of :eyes: ?

Reply:
If you wanna run this locally, comment out this line and try to wrap on rinkeby.